lms/templates/student_account/emails/email_change_confirmation/message_body.txt

```diff
@@ -14,6 +14,7 @@ ${_("You successfully changed the email address associated with your "
     )
 }
 
-## Farewell
-${_("Thanks,")}
-${_("- The edX Team")}
+## Translators: This is the signature of an email
```

Comment:
Indicate that `\n` is a new line character and should be placed between the closing word and the signing team's name.